packages/notifi-react/lib/components/HistoryDetail.tsx

```diff
@@ -58,11 +59,11 @@ export const HistoryDetail: React.FC<HistoryDetailProps> = (props) => {
       >
         <div
           className={clsx(
-            'notifi-history-detail-subject',
-            props.classNames?.subject,
+            'notifi-history-detail-topic',
+            props.classNames?.topicName,
           )}
         >
-          {historyItem.subject}
+          {historyItem.topic}
```

Comment:
I guess the must needed change is this line. 
Do we have to change other lines? ex. changing classNames override & css class.

Reply:
Yes, that’s correct. I wanted to make sure the props aligned accurately with the content, but I’ve reverted the other changes to keep it focused on the essential update and minimize impact.